basic-data-url.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Model: basic DataURL test</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Model: Basic DataURL test overriding type</h1>
+    <hr />
+
+    <h2>HTML</h2>
+    <pre>
+&lt;model alt="A model of a teapot" data-src="./teapot.usdz" data-type="model/vnd.usdz+zip">
+&lt;/model>
+    </pre>
+    <h2>JavaScript</h2>
+<pre>
+const model = document.querySelector('model');
+const {src, type} = model.dataset;
+const bytes = await (await fetch(src)).bytes();
+const srcString = URL.createObjectURL(new Blob([bytes], {type}));
```

Comment:
This is actually a blob URL test case, as using createObjectURL => blob.

Blob URL:
 ```js
const response = await fetch(url);
const blob = await response.blob(); // this should maintain type too
const objectURL = URL.createObjectURL(blob);
```

We should probably have two demo pages, one for data URL, and blob URL.

~data URL:
```js
const response = await fetch(url);
const blob = await response.blob();
const reader = new FileReader();
reader.readAsDataURL(blob);
```